src/controllers/sound/SoundController.cpp

```diff
@@ -0,0 +1,195 @@
+#include "controllers/sound/SoundController.hpp"
+
+#include "common/QLogging.hpp"
+#include "debug/Benchmark.hpp"
+#include "singletons/Paths.hpp"
+#include "singletons/Settings.hpp"
+
+#define MINIAUDIO_IMPLEMENTATION
+#include <miniaudio.h>
+
+#include <limits>
+#include <memory>
+
+namespace chatterino {
+
+constexpr const auto NUM_SOUNDS = 4;
+
+SoundController::SoundController()
+    : context(std::make_unique<ma_context>())
+    , resourceManager(std::make_unique<ma_resource_manager>())
+    , device(std::make_unique<ma_device>())
+    , engine(std::make_unique<ma_engine>())
+{
+}
+
+// TODO: How do we know what the default ping sound is?
```

Comment:
Remove this comment

Reply:
Resolved in [`bed44d1` (#4334)](https://github.com/Chatterino/chatterino2/pull/4334/commits/bed44d18a311db365d0b706dc93ec32e3564f69f)